src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -52,12 +52,14 @@
             transition: box-shadow $container-transition-duration ease $container-transition-delay;
 
             &.bcs-is-focused {
-                box-shadow: 0 0 $bdl-grid-unit * 4 $bdl-gray-30;
-                animation: bcs-ActivityItem-callout 1s ease-in-out $container-transition-delay 0 normal none; // Disabled by default
-
-                &:not(:focus-within) {
-                    animation-iteration-count: 1; // Only play the animation if the user is not interacting directly with the element or its descendents
-                    animation-fill-mode: forwards;
+                .bcs-ActivityCard {
+                    box-shadow: 0 0 $bdl-grid-unit * 4 $bdl-gray-30;
+                    animation: bcs-ActivityItem-callout 1s ease-in-out $container-transition-delay 0 normal none; // Disabled by default
+
+                    &:not(:focus-within) {
```

Comment:
:focus-within isn't supported on IE, but this looks non-critical (and is already in place)